package.json

```diff
@@ -23,26 +23,26 @@
     "build:dummy-wallet-app": "pnpm turbo build --filter=\"@ledgerhq/dummy-wallet-app\"",
     "build:dummy-live-app": "pnpm turbo build --filter=\"@ledgerhq/dummy-live-app\"",
     "build:actions": "pnpm turbo build --filter=@actions/*",
-    "build:cli": "pnpm turbo build --no-daemon --filter=live-cli",
+    "build:cli": "pnpm turbo build --no-daemon --filter=@ledgerhq/live-cli",
     "build:coin": "pnpm turbo build --no-daemon --filter=\"./libs/coin-**\"",
-    "build:domain": "pnpm turbo build --no-daemon --filter=domain-service",
-    "build:network": "pnpm turbo build --no-daemon --filter=live-network",
-    "build:config": "pnpm turbo build --no-daemon --filter=live-config",
-    "build:llc": "pnpm turbo build --no-daemon --filter=\"live-common\"",
+    "build:domain": "pnpm turbo build --no-daemon --filter=@ledgerhq/domain-service",
+    "build:network": "pnpm turbo build --no-daemon --filter=@ledgerhq/live-network",
+    "build:config": "pnpm turbo build --no-daemon --filter=@ledgerhq/live-config",
+    "build:llc": "pnpm turbo build --no-daemon --filter=./libs/ledger-live-common",
     "build:lld": "pnpm turbo build --no-daemon --filter=ledger-live-desktop",
```

Comment:
Looks like we are missing some namespacing in these scripts - I see a few examples here

Reply:
(for example, the next 4 scripts after this one, but many more further down in this file)

Reply:
Do you mean:
```
    "build:lld:deps": "pnpm turbo build --no-daemon --filter=\"ledger-live-desktop^...\"",
    "build:llm:android": "pnpm turbo android:apk:local --no-daemon --filter=live-mobile",
    "build:llm:ios": "pnpm turbo ios:local:ipa --no-daemon --filter=live-mobile",
    "build:llm:deps": "pnpm turbo build --no-daemon --filter=\"live-mobile^...\"",
```
They are correct and work as expected as we don't have the `@ledgerhq` prefix for theses.

Reply:
Yes that's what I mean, but I am surprised it works. The upgrade guide notes that `--filter` no longer infers namespaces and links to [this PR](https://github.com/vercel/turborepo/pull/8137) that removes that functionality. In that PR, they say that unit tests have been updated "to verify inference no longer works and using the explicit package name still works".

I'm confused that this works in this case (any ideas why?), and am still hesitant that we use this pattern if it's now unsupported?

Reply:
@angusbayley  Otherwise it will  `errors when no package names or directories are matched`, since `ledger-live-desktop`and `live-mobile`are directories on the same project, turbo will find them. and that's why it's working.

Reply:
turbo does not need to infer as `ledger-live-desktop` and `live-mobile` are the package name

Reply:
Ah okay - the names threw me. Thanks guys.

Do you know why there's inconsistency in the names? *ledger-*live-desktop and live-mobile

Reply:
(resolving this because it should no longer block the PR)